### DISCUSSION TITLE
Time unit 86400 meaning in the yaml file

### DISCUSSION THREAD

**Discussion by abdulrehman565** [Deltares/hydromt#360] [Q&A]
I have two questions:
1.  In the data_catalog.yaml file, What does time:86400 mean in unit add?
hirps_global:
  crs: 4326
  data_type: RasterDataset
  driver: netcdf
  meta:
    category: meteo
    paper_doi: 10.3133/ds832
    paper_ref: Funk et al (2014)
    source_license: CC
    source_url: https://www.chc.ucsb.edu/data/chirps
    source_version: v2.0
  path: chirps_global.nc
  **unit_add:
    time: 86400**
    (the bold part)

2. I am currently using era5_land precipitation data. And this product accumulates all the precipitation values in the 0th hour so I'm using only the 0th hour data for a number of years. When I ran the model, the result came out to be wrong. How to use the precipitation netcdf file for 0th hour only?

**Comment by Safa136** [comment]
I think 86400 is converting 1 day time step to seconds (1 * 24* 60*60).
